vault/resource_database_secret_backend_connection_test.go

```diff
@@ -757,6 +757,44 @@ func TestAccDatabaseSecretBackendConnection_mysql_tls(t *testing.T) {
 	})
 }
 
+func TestAccDatabaseSecretBackendConnection_mysql_aurora_tls(t *testing.T) {
```

Comment:
I don't think we need a separate test for this, we can add this as an additional test step to the test above this on L755, `TestAccDatabaseSecretBackendConnection_mysql_tls`. We just need to add a test step for MySQL Aurora, something like the following:

```
			{
				Config: testAccDatabaseSecretBackendConnectionConfig_mysql_aurora_tls(name, backend, connURL, password, tlsCA, tlsCertificateKey),
				Check: testComposeCheckFuncCommonDatabaseSecretBackend(name, backend, pluginName,
					resource.TestCheckResourceAttr(testDefaultDatabaseSecretBackendResource, "allowed_roles.#", "2"),
					resource.TestCheckResourceAttr(testDefaultDatabaseSecretBackendResource, "allowed_roles.0", "dev"),
					resource.TestCheckResourceAttr(testDefaultDatabaseSecretBackendResource, "allowed_roles.1", "prod"),
					resource.TestCheckResourceAttr(testDefaultDatabaseSecretBackendResource, "root_rotation_statements.#", "1"),
					resource.TestCheckResourceAttr(testDefaultDatabaseSecretBackendResource, "root_rotation_statements.0", "FOOBAR"),
					resource.TestCheckResourceAttr(testDefaultDatabaseSecretBackendResource, "verify_connection", "true"),
					resource.TestCheckResourceAttr(testDefaultDatabaseSecretBackendResource, "mysql_aurora.0.connection_url", connURL),
					resource.TestCheckResourceAttr(testDefaultDatabaseSecretBackendResource, "mysql_aurora.0.max_open_connections", "2"),
					resource.TestCheckResourceAttr(testDefaultDatabaseSecretBackendResource, "mysql_aurora.0.max_idle_connections", "0"),
					resource.TestCheckResourceAttr(testDefaultDatabaseSecretBackendResource, "mysql_aurora.0.max_connection_lifetime", "0"),
					resource.TestCheckResourceAttr(testDefaultDatabaseSecretBackendResource, "data.%", "1"),
					resource.TestCheckResourceAttr(testDefaultDatabaseSecretBackendResource, "data.password", password),
					resource.TestCheckResourceAttr(testDefaultDatabaseSecretBackendResource, "mysql_aurora.0.tlsCA", tlsCA+"\n"),
					resource.TestCheckResourceAttr(testDefaultDatabaseSecretBackendResource, "mysql_aurora.0.tls_certificate_key", tlsCertificateKey+"\n"),
				),
			},

```

This way we don't have to do the same setup and read the environment variables again through a separate test, and can reuse the same setup as the existing test step :)

Reply:
addressed. tests work!